rdflib/parser.py

```diff
@@ -181,23 +181,35 @@ def __init__(self, system_id=None, format=None):
 
         # copy headers to change
         myheaders = dict(headers)
-        if format == "application/rdf+xml":
+        if format == "xml":
             myheaders["Accept"] = "application/rdf+xml, */*;q=0.1"
         elif format == "n3":
             myheaders["Accept"] = "text/n3, */*;q=0.1"
-        elif format == "turtle":
-            myheaders["Accept"] = "text/turtle,application/x-turtle, */*;q=0.1"
+        elif format in ["turtle" or "ttl"]:
```

Comment:
The expression `["turtle" or "ttl"]` always evaluates to `['turtle']`.

```python
>>> ["turtle" or "ttl"]
['turtle']
```

Should it be `["turtle", "ttl"]`?

Reply:
Ooops, thganks @edmondchuc! You nailed it.